pkg/defkinds/nodejs/builder_test.go

```diff
@@ -41,6 +44,33 @@ func initBuildLLBForDevStageTC(t *testing.T, mockCtrl *gomock.Controller) buildT
 			ContextName:   "context",
 		},
 		expectedState: "testdata/build/state-dev.json",
+		expectedImage: &image.Image{
+			Image: specs.Image{
+				Architecture: "amd64",
+				OS:           "linux",
+				RootFS: specs.RootFS{
+					Type: "layers",
+				},
+			},
+			Config: image.ImageConfig{
+				ImageConfig: specs.ImageConfig{
+					User: "1000",
+					Env: []string{
+						"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+						"NODE_ENV=development",
+					},
+					// Following entrypoint is automatically defined by the
+					// base image. Maybe it should not be kept? :thinking:
```

Comment:
I think it's OK to keep default base image settings :thinking: . People may be used to it.